profiles/sssd/password-auth

```diff
@@ -15,22 +16,25 @@ auth        required                                     pam_deny.so
 
 account     required                                     pam_access.so                                          {include if "with-pamaccess"}
 account     required                                     pam_faillock.so                                        {include if "with-faillock"}
+-account    sufficient                                   pam_systemd_home.so                                    {include if "with-homed"}
```

Comment:
@poettering Shouldn't this go after pam_unix?

Reply:
so, homed synthesizes NSS records including shadow records from its native JSON user records, which include the account mgmt settings. These are more expressive than the NSS ones (a true superset), and can give more precise error messages. Hence it's kinda essential to run pam_systemd_home first, so that it can do its thing and then shortcut the PAM stack, so that the pam_unix.so stuff doesn't run.

If you'd run pam_unix first, then it would run the checks first, possibly output some stuff, and then pam_systemd_homed would check things again, possibly find more finegrained restrictions and output the same info, but more precise. So you'd end up with duplicate output, but one is more precise then the other.

hence, this order should be OK. There's no loss of validation checks, but more friendly output if done this way (i.e. less duplication and more precise info)